src/Features/header/components/MobileApplicationPromo/MobileApplicationPromo.tsx

```diff
@@ -0,0 +1,80 @@
+import { FaGooglePlay, FaApple } from 'react-icons/fa';
+import { AppButton } from '../../../../components/AppButton';
+import { ReactIcon } from '../../../../components/ReactIcon';
+
+export const MobileApplicationPromo = () => {
```

Comment:
Set return of this component to `ReactElement` from react. Please, fix all components in this PR